src/nupic/bindings/algorithms.i

```diff
@@ -636,91 +636,6 @@ void forceRetentionOfImageSensorLiteLibrary(void) {
 }
 
 //--------------------------------------------------------------------------------
-// LearningSet for continuous FDR TP
```

Comment:
It looks like we're removing methods that are no longer necessary, which is the main change in this PR, right?